install-dev/langs/en/data/tab.xml

```diff
@@ -96,7 +96,7 @@
     <tab id="Products_1" name="Product Settings" />
     <tab id="Products" name="Products" />
     <tab id="Products_v2" name="Products (experimental)" />
-    <tab id="Profiles" name="Profiles" />
+    <tab id="Profiles" name="Roles" />
```

Comment:
```suggestion
    <tab id="Roles" name="Roles" />
```

Reply:
Is this necessary to change ID? PHP class is still called `Profiles`.